src/FileProviders/Physical/test/PhysicalFileProviderTests.cs

```diff
@@ -1378,6 +1378,63 @@ public async Task WildCardToken_RaisesEventsWhenFileSystemWatcherDoesNotFire()
             }
         }
 
+        [Fact]
+        public void UsePollingFileWatcher_FileWatcherNull_SetsSuccessfully()
+        {
+            // Arrange
+            using (var root = new DisposableFileSystem())
+            {
+                using (var provider = new PhysicalFileProvider(root.RootPath))
+                {
+                    Assert.False(provider.UsePollingFileWatcher);
+
+                    // Act / Assert
+                    provider.UsePollingFileWatcher = true;
+                    Assert.True(provider.UsePollingFileWatcher);
+                }
+            }
+        }
+
+        [Fact]
+        public void UsePollingFileWatcher_FileWatcherNotNull_SetterThrows()
+        {
+            // Arrange
+            using (var root = new DisposableFileSystem())
+            {
+                using (var fileSystemWatcher = new MockFileSystemWatcher(root.RootPath))
+                {
+                    using (var physicalFilesWatcher = new PhysicalFilesWatcher(root.RootPath + Path.DirectorySeparatorChar, fileSystemWatcher, pollForChanges: false))
+                    {
+                        using (var provider = new PhysicalFileProvider(root.RootPath) { FileWatcher = physicalFilesWatcher })
+                        {
+                            // Act / Assert
+                            Assert.Throws<InvalidOperationException>(() => { provider.UsePollingFileWatcher = true; });
+                        }
+                    }
+                }
+            }
+        }
+
+        [Fact]
+        public void UsePollingFileWatcher_FileWatcherNotNull_GetterEfzUp()
```

Comment:
`EfzUp` ??

Reply:
Please ignore. I thought I removed that.